veracruz-client/src/veracruz_client.rs

```diff
@@ -340,7 +308,7 @@ impl VeracruzClient {
                 if self.policy_hash != received_hash {
                     return Err(VeracruzClientError::MismatchError {
                         variable: "check_pi_hash",
-                        expected: self.policy.pi_hash().clone().into_bytes(),
+                        expected: b"check policy file".to_vec(),
```

Comment:
Does the type of the `VeracruzClientError::MismatchError` need to change?  What is the value of the `expected` field, here?